lib/tracker_api/resources/story.rb

```diff
@@ -21,7 +21,7 @@ class Story
       attribute :integration_id, Integer
       attribute :kind, String
       attribute :label_ids, [Integer]
-      attribute :labels, [Label]
+      attribute :labels, [Label], default: nil
```

Comment:
@whysthatso does this make sense?

I am actually working on another branch that will remove the existing dirty tracking. It has caused way more issues than it is worth. IMO.